app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -364,6 +364,7 @@ public void onCreate(Bundle savedInstanceState) {
     searchFiles();
     tabRecyclerView.setAdapter(tabsAdapter);
     new ItemTouchHelper(tabCallback).attachToRecyclerView(tabRecyclerView);
+    drawerLayout.setDescendantFocusability(ViewGroup.FOCUS_BLOCK_DESCENDANTS);
```

Comment:
Is it possible to move this to xml?

Reply:
@abdulwd Yes, that can be done. Just wanted to know what difference will it make?